src/overlays/actors/ovl_Door_Shutter/z_door_shutter.c

```diff
@@ -682,6 +682,13 @@ void DoorShutter_Draw(Actor* thisx, GlobalContext* globalCtx) {
         s32 pad[2];
         ShutterInfo* sp70 = &D_80998134[this->unk_16C];
 
+        //! @bug In very niche scenarios, depending on the placement of other doors when a new room is loading,
```

Comment:
Is it possible/worth it to be any more specific here, especially if it was so hard to figure out in the first place?

Reply:
yea I didnt want the comment to be a novel lmao
I think I got the point across though, if theres a door placed such that it will be drawing the same frame it spawns (a door nearby where you are entering from another door), the issue can occur. Thats what I mean by `depending on the placement of other doors when a new room is loading`

Reply:
We could get into more detail about how the doors need to be placed, but really any new project should just patch this asap and not worry about it. Cause it all depends on the placement of thing in the map

Reply:
Yeah I kind of agree since when you explained this to me earlier I was still a little confused why this happens at all, but I'd just emphasize that the displaylist index not being set is what causes the bug

Reply:
I think the more information we have about a bug, the better, even if everybody will just patch the bug.

Reply:
So does it happen when it's placed so that the camera sees it too early (and so demands Draw before Update happens for the first time), is that the point?

Reply:
Yes, if it is not culled the same frame it is spawned (init has just run), then it will draw once before update 

How this issue doesn't happen in vanilla, idk. In this rom hack I mentioned, the door is shaped like a U, and the exit is directly in line of sight behind the player during the door closing animation.

Reply:
Ah, in that case I think something like "the door is placed so that the camera is on it in the first frame after init" or "placed so that it needs to be drawn too soon" or similar would cover it without going overboard. Just to be more specific about exactly what about the placement causes the issue.

Reply:
After spending some time talking to Fig and understanding the glitch more, I think the comment should be something like this, positioned above the IF statement:
 
            This actor isn't fully initialized until the first time Update is called when the actor's dependent
            object is loaded. The check objBankIndex == requiredObjBankIndex stops the actor from drawing until Update 
            completes initialization, except in the case when requiredObjBankIndex is OBJECT_GAMEPLAY_KEEP.
            In this case, unk_16C will be 0, and try to draw the display list of the first entry in `D_80998134`, which can crash 
            the game.